composer/trainer/trainer.py

```diff
@@ -654,10 +623,10 @@ class Trainer:
             artifact stores.
         autoresume (bool, optional): Whether or not to enable autoresume, which allows for stopping and resuming
             training. This allows use of spot instances, as the training run is now fault tolerant.  This parameter
-            requires ``checkpoint_save_path`` and ``run_name`` to be specified and ``save_overwrite`` to be ``False``.
+            requires ``checkpoint_save_path`` and ``run_name`` to be specified.
             (default: ``False``)
 
-            When enabled, the checkpoint_save_path is checked for checkpoints of the format ``"{checkpoint_save_path}/{save_latest_filename}"``,
+            When enabled, the checkpoint_save_path is checked for checkpoints of the format ``"{checkpoint_save_path}/{latest_checkpoint_filename}"``,
```

Comment:
nit: `latest_checkpoint_filename` doesn't look like an argument to the trainer, so this seems confusing to the user as part of the docs?

Reply:
done